tools/lsp/common.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use i_slint_compiler::CompilerConfiguration;
+use std::path::{Path, PathBuf};
+
+use crate::ServerNotifier;
+
+// Common data structures:
```

Comment:
Make it a module doc comment.

Common between what and what?  (add that in the module comment)